page-objects/administration.js

```diff
@@ -67,6 +69,15 @@ submitConsent: async function(e_mail) {
         }
     }
 },
+checkIfElementIsVisisble: async function (itemToCompare) {
```

Comment:
elements to be visible should be part of the parameters and generically work

Reply:
I've added second parameter to function.